ui/packages/components/src/RunDetailsV2/ErrorCard.tsx

```diff
@@ -0,0 +1,31 @@
+import { Alert } from '@inngest/components/Alert';
+import { NewButton } from '@inngest/components/Button';
+
+type Props = {
+  error: Error | any;
+  reset?: () => void;
+};
+
+export function ErrorCard({ error, reset }: Props) {
```

Comment:
Should be it's own component once part of the design system. Had to re-use this as the error page is only in dashboard for now. Forgive my sins.